docs/developer/serving/traffic-management.md

```diff
@@ -0,0 +1,220 @@
+# Traffic management
+
+You can manage traffic routing to different Revisions of a Knative Service by modifying the `traffic` spec of the Service resource.
+
+When you create a Knative Service, it does not have any default `traffic` spec settings. By setting the `traffic` spec, you can split traffic over any number of fixed Revisions, or send traffic to the latest Revision by setting `latestRevision: true` in the spec for a Service.
+
+## Traffic routing examples
+
+The following example shows a `traffic` spec where 100% of traffic is routed to the `latestRevision` of the Service. Under `status` you can see the name of the latest Revision that `latestRevision` was resolved to:
+
+```yaml
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: example-service
+  namespace: default
+spec:
+...
+  traffic:
+  - latestRevision: true
+    percent: 100
+status:
+  ...
+  traffic:
+  - percent: 100
+    revisionName: example-service-1
+```
+
+The following example shows a `traffic` spec where 100% of traffic is routed to the `current` Revision, and the name of that Revision is specified as `example-service-1`. The latest ready Revision is kept available, even though no traffic is being routed to it:
+
+```yaml
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: example-service
+  namespace: default
+spec:
+...
+  traffic:
+  - tag: current
+    revisionName: example-service-1
+    percent: 100
+  - tag: latest
+    latestRevision: true
+    percent: 0
+```
+
+The following example shows how the list of Revisions in the `traffic` spec can be extended so that traffic is split between multiple Revisions. This example sends 50% of traffic to the `current` Revision, `example-service-1`, and 50% of traffic to the `candidate` Revision, `example-service-2`:
+
+```yaml
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: example-service
+  namespace: default
+spec:
+...
+  traffic:
+  - tag: current
+    revisionName: example-service-1
+    percent: 50
+  - tag: candidate
+    revisionName: example-service-2
+    percent: 50
+  - tag: latest
+    latestRevision: true
+    percent: 0
+```
+
+## Routing and managing traffic by using the Knative CLI
+
+You can use the following `kn` CLI command to split traffic between revisions:
+
+```bash
+kn service update <service-name> --traffic <revision-name>=<percent>
```

Comment:
We should add an example here also how to set a tag for a revision name, and the usage of `@latest` to configure the entry for the `latestRevision: true` entry.

For example

```
# Add a tag green and blue  (blue is pointing to the `latest: true` entry)
kn service update example --tag green=revision-0001 --tag blue=@latest

# Set a traffic split with 80% to latest and 20% to "v1"
kn service update example --traffic @latest=80 --traffic v1=20
```